views/snippets/form_date.html

```diff
@@ -21,7 +21,7 @@ <h1 class="heading-small">What is your date of birth?</h1>
 
       <div class="form-group form-group-year">
         <label for="dob-year">Year</label>
-        <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="0" max="2014">
+        <input class="form-control" id="dob-year" name="dob-year" type="number" pattern="[0-9]*" min="1910" max="2014">
```

Comment:
It would do. Ideally it would also accept two digit entries, but like Leéonie said this will result in a many key presses for AT users. 

@aduggin do you have a preference here?